### DISCUSSION TITLE
How to set minimum width for system tray window?

### DISCUSSION THREAD

**Discussion by billywatkins** [tauri-apps/tauri#8518] [Q&A] [answered]
Hello, I am trying to set the minimum width for the system tray window, but I don't know how.

```rust
use tauri::{CustomMenuItem, SystemTray, SystemTrayMenu, SystemTrayMenuItem};

pub fn setup_system_tray() -> SystemTray {
    let open = CustomMenuItem::new("open".to_string(), "open");
    let tray_menu = SystemTrayMenu::new()
        .add_item(open)
    SystemTray::new().with_menu(tray_menu)
}

```

**Comment by FabianLars** [comment, marked as answer]
Tauri doesn't have such a setting and i've never heard of it before to be honest. Do you know of any other gui framework that has a config like this for _native_ tray menus that we could look at to see how we could implement it?

**Reply by billywatkins** [reply]
I am completely new to Mac development. I did some research, but I couldn't find any information.

**Reply by FabianLars** [reply]
Same (also somewhat new to mac and couldn't find anything). I couldn't find anything for windows either.
